carddrawer/src/main/res/layout/card_drawer_bottom_label.xml

```diff
@@ -24,10 +24,13 @@
         android:visibility="invisible"
         tools:visibility="visible"
         android:textColor="@color/card_drawer_ui_white"
-        android:textSize="@dimen/card_drawer_xxxs_text"
```

Comment:
Is it ok to remove this?

Reply:
I assume it is, because the text size is set by the function 'getTextPixelSize' on 'BottomLabel.kt' (Line 89). I made a screenshots comparison using that 'textSize' property and not using it and I didn't see any difference.